pytest_bdd/steps.py

```diff
@@ -220,6 +220,8 @@ def set_code(func, code):
     ]
     if six.PY3:
         argnames.insert(1, "co_kwonlyargcount")
+        if sys.version_info.minor >= 8:
+            argnames.insert(1, "co_posonlyargcount")
```

Comment:
I'm checking the python documentation for code objects, and it seems that `co_kwonlyargcount` is there at least since python 3.5: https://docs.python.org/3.5/library/inspect.html?highlight=co_kwonlyargcount.

Does the test suite break if we don't do the `sys.version_info.minor >= 8` check?